pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -159,7 +159,7 @@ var (
 	// creating flakes in unsupported versions or rediscovering
 	// already-fixed bugs. This variable should be updated periodically
 	// as releases reach end of life.
-	OldestSupportedVersion = clusterupgrade.MustParseVersion("v22.2.0")
+	OldestSupportedVersion = clusterupgrade.MustParseVersion("v23.1.0")
```

Comment:
Nit: we already have a notion of `MinSupportedVersion` for unit tests; the above being kinda isomoprhic, maybe we could keep the same terminology?

Reply:
The semantics are slightly different though. In cockroachdb/tests, `MinSupportedVersion` is the lowest version a binary should remain compatible with. On master, that's currently `23.2` and it's different for each release branch.

`OldestSupportedVersion` is the oldest version of cockroachdb still officially supported (i.e., if there are bugs found in that release, we'll fix them). This is release branch agnostic and currently 23.1.

Reply:
Thanks, it wasn't immediately clear from the comment. It might be worth adding the above to highlight the difference (wrt MinSupportedVersion).

Reply:
Good point, updated.